docs/administer/api_rbac.md

```diff
@@ -0,0 +1,40 @@
+# Navigating the API changes for RBAC
+
+Starting with Percona Everest v1.2.0, breaking changes are being made to the API for monitoring instances and backup storage resources. These changes are:
+
+- Before the release of Percona Everest 1.2.0, these resources were globally scoped, but now they will be specific to namespaces. 
+
+- The database clusters will only be able to use monitoring instances and backup storages that are located within the same namespace as the cluster. The system used a `.spec.allowedNamespaces` field to control access to these global resources. This field determined the namespaces where the resource could be accessed, providing a certain degree of access control.
+
+- With the update to Percona Everest v1.2.0, the shift from global scoping to namespacing these resources marks a significant change in how access control is managed, enhancing security by ensuring that resources are only accessible within their designated namespaces.
+
+## Challenges of globally spaced namespaces
```

Comment:
TBH this heading doesn't make much sense to me. 😅 

Please consider something like:
- Challenges with globally scoped resources
- Why?

Reply:
Fixed.

Reply:
I still see the older title `Challenges of globally spaced namespaces`

Reply:
Fixed